include/abb_librws/rws_interface.h

```diff
@@ -512,6 +512,16 @@ class RWSInterface
    */
   bool endSubscription();
 
+  /**
+   * \brief Close the active subscription connection.
+   *
+   * This will cause waitForSubscriptionEvent() to return or throw.
+   * It does not delete the subscription from the controller.
+   *
+   * This function blocks until an active waitForSubscriptionEvent() has finished.
+   */
+  void closeSubscription();
```

Comment:
```suggestion
  void forceCloseSubscription();
```
Slightly more descriptive name.